dnn/variational_with_real_data.ipynb

```diff
@@ -7,23 +7,23 @@
         "colab_type": "text"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    animate(dnn2_means, 1, 1)
</pre><p>Let's make sure not to include plots now that this is public</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tnc-br/ddf-isoscapes/pull/155/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
Ack, done.